src/sonic-host-services/scripts/hostcfgd

```diff
@@ -643,9 +852,8 @@ class HostConfigDaemon:
         self.hostname_cache=""
         self.aaacfg = AaaCfg()
         self.iptables = Iptables()
+        self.feature = FeatureHandler(self.config_db, self.device_config)
```

Comment:
Maybe rename this `self.feature_handler`?

Reply:
Done